### ISSUE TITLE
Need to only force custom inset when iPhone X/XS, not for all devices

### ISSUE THREAD

**Issue by ktownsend-cbre** [react-native-community/react-native-safe-area-view#48]
It would be much more useful to me if the forceInset overrides were only utilized when the phone is iPhone X/XS, not simply applied regardless of the device. In my case, I want to override the insets with custom values, but only for iPhone X/XS. All other devices shouldn't add any padding. Either the forceInset logic should be skipped when not iPhone X/XS, or a parallel prop of forceInsetOnlyX could be used to apply the inset overrides conditionally without breaking the existing behavior of forceInset. 

I can do the conditional when specifying my forceInsert values, but I will have to replicate detection of iPhoneX/XS because your component doesn't expose that detection result (would be nice if you did).

**Comment by SebastianKurp** [comment]
A little late to the party... but one way you can do is to create a specific profile for an iPhone X. 
```
if(screenHeight && screenWidth matches iPhone X's dimensions) {
   return isIphoneX = true
}
```
Then just create a custom component using SafeAreaView that has logic to handle what you want to do with iPhone Xs.

**Comment by dustinbahr** [comment]
Has this not been solved yet?

In my case I have a hidden status bar. 

So, on normal devices I don't want any inset. But on iPhones with the notch (iPhone X, etc), I still need it to inset, because of the notch, instead of because of the actual status bar info.

This seems like a normal use case.

**Comment by brentvatne** [comment]
this should be resolved in version 1.0 which was released today, thanks for reporting!

**Comment by ktownsend-cbre** [comment]
Thanks @brentvatne
